portfolio new/index.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="apple-touch-icon" sizes="180x180" href="./Favicon/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="./Favicon/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="./Favicon/favicon-16x16.png">
+<link rel="manifest" href="./Favicon/site.webmanifest">
+    <!-- bootstrap css -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  
+       <!-- fonts -->
+       <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Nunito&family=Satisfy&display=swap" rel="stylesheet">
+  <!-- my css -->
+  <link rel="stylesheet" href="css/style.css"> 
+<!-- title -->
+    <title>Student | Portfolio</title>
+</head>
+<body>
+  <!-- navbar -->
+  <main-header>
+    
```

Comment:
add a link here in your home tab to /index.html for ease of navigation back to repo landing page